### ISSUE TITLE
Differentiate from "trending because whale" and "trending because a lot of views"

### ISSUE THREAD

**Issue by seanyesmunt** [lbryio/lbry-desktop#3544]
From EF on discord:
> Idea from @Jorge Clavellina LBRY channel after we had lots of convo about trending on LBRY Social server. It is to put a little "boosted" badge next to claims if you're in trending view and a claim has more than, say, 5000 LBC in support. That way, people will understand why low-view content is up there sometimes (okay, a lot).

**Comment by kauffj** [comment]
This would just encourage them to make 50 transactions instead.

**Comment by eggplantbren** [comment]
Yeah, but that has a cost. Any interest in my PR? It doesn't solve this but does somewhat improve it IMHO, due to massive diminishing marginal returns to 1 extra LBC.

Some more ideas to encourage casual viewers to support content will help too, I think.

* Enable it by default instead of having it in experimental settings.
* Have an option in the app to auto-abandon the support after 24 hours. That could be hard depending on where you implement it, but it would make supporting very convenient.

**Comment by kauffj** [comment]
@eggplantbren your PR is supposed to be merged, I assume it got snagged on something in the sausage factory.

Those are all interested suggestions. Enabling supports for everyone is happening soon (UI needs a bit of a touch up).

Something larger will be coming as well, but it is going to take at least a few weeks.

**Comment by eggplantbren** [comment]
I would love to know what the something larger is. Feel free to DM if it's a secret but not that much of a secret.

**Comment by kauffj** [comment]
Recommended reading: https://vitalik.ca/general/2019/04/03/collusion.html
